packages/edit-site/src/components/global-styles/font-families.js

```diff
@@ -46,11 +39,6 @@ function FontFamilies() {
 		: [];
 	const hasFonts = 0 < customFonts.length || 0 < themeFonts.length;
 
-	const hasInstalledFonts =
-		hasFonts ||
-		baseFontFamilies?.theme?.length > 0 ||
-		baseCustomFonts?.length > 0;
```

Comment:
There may be cases where fonts are installed in a theme, but all fonts are disabled. In this case, the expected text is "No fonts activated" and the button text is "Manage fonts".

However, if we remove these codes, including `hasInstalledFonts`, the expected text does not appear:

![fonts](https://github.com/user-attachments/assets/72956fbd-6173-41ea-90d8-5b38850d6058)

See #63533 for the intention of these codes.

I don't think this PR should make any changes other than reverting the font grouping.